cmd/snap/cmd_version_test.go

```diff
@@ -42,7 +42,12 @@ func (s *SnapSuite) TestVersionCommandOnClassic(c *C) {
 
 	_, err := snap.Parser(snap.Client()).ParseArgs([]string{"version"})
 	c.Assert(err, IsNil)
-	c.Assert(s.Stdout(), Equals, "snap    4.56\nsnapd   7.89\nseries  56\nubuntu  12.34\n")
+	c.Assert(s.Stdout(), Equals, ""+
+		"snap          4.56\n"+
+		"snapd         7.89\n"+
+		"series        56\n"+
+		"ubuntu        12.34\n"+
+		"architecture  ia64\n")
```

Comment:
I had an itanium CPU as a bookshelf terminator for a while. The pure-copper heatsinks were very heavy.